docs/guide/tutorials/aggregation.txt

```diff
@@ -1,3 +1,137 @@
 ===========
 Aggregation
 ===========
+
+Overview
+--------
+
+Aggregation operations process data records and return
+computed results. Aggregation operations group values from
+multiple documents together, and can perform a variety of
+operations on the grouped data to return a single result.
+
+The Aggregation Pipeline
+------------------------
+
+The aggregation pipeline is a framework for data aggregation
+modeled on the concept of data processing pipelines. Documents
+enter a multi-stage pipeline that transforms the documents into
+aggregated results.
+
+For a full explanation and a complete list of pipeline stages
+and operators, see the :manual:`manual </core/aggregation-pipeline/>`
+
+The following example uses the aggregation pipeline on the
+`restaurant <https://docs.mongodb.org/getting-started/node/import-data/>`_ 
+sample dataset to find a list of restaurants located in the Bronx,
+grouped by restaurant category.
+
+.. code-block:: js
+
+   const { MongoClient } = require('mongodb');
+   
+   // Connection URL
+   const url = 'mongodb://localhost:27017';
+   
+   // Create a new MongoClient
+   const client = new MongoClient(url);
+
+   async function main(client) {
+     const collection = client.db('myproject').collection('restaurants');
+
+     const cursor = collection.aggregate([
+       { $match: { borough: 'Bronx' } },
+       { $unwind: '$categories'},
+       { $group: { _id: '$categories', Bronx restaurants: { $sum: 1 } } }
```

Comment:
not sure what this line is intended to be, but `{ $group: { _id: '$categories', Bronx restaurants: { $sum: 1 } } }` isn't valid json. The `Bronx restaurants` here is going to cause an error

Reply:
Ah, that was meant to be `'Bronx restaurants'`. Pushing fix